bitnami/kubernetes-event-exporter/values.yaml

```diff
@@ -1,93 +1,121 @@
+##
+nameOverride:
+## @param fullnameOverride String to fully override kubernetes-event-exporter.fullname template
+##
+fullnameOverride:
+## @param commonAnnotations Annotations to add to all deployed objects
 ##
 commonAnnotations: {}
-
-## Common labels to add to all resources. Evaluated as a template
+## @param commonLabels Labels to add to all deployed objects
 ##
 commonLabels: {}
+## @param extraDeploy Array of extra objects to deploy with the release (evaluated as a template)
+##
+extraDeploy: []
 
-## Deployment pod host aliases
+## @section Kubernetes Event Exporter parameters
+
+image:
+  ## @param image.registry Container image registry
+  ## @param image.repository Container image name
+  ## @param image.tag Container image tag
+  ## @param image.pullPolicy Container image pull policy
+  ## @param image.pullSecrets Specify docker-registry secret names as an array
+  ##
+  registry: docker.io
+  repository: bitnami/kubernetes-event-exporter
+  tag: 0.10.0-debian-10-r0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ## Example:
+  ## pullSecrets:
+  ##   - myRegistryKeySecretName
+  ##
+  pullSecrets: []
+## @param hostAliases Add deployment host aliases
 ## https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/
 ##
 hostAliases: []
-
+## @param config.logLevel Verbosity of the logs (options: `fatal`, `error`, `warn`, `info` or `debug`)
+## @param config.logFormat How the logs are formatted. Allowed values: `pretty` or `json`
+## @param config.receivers [array] Array containing event receivers
+## @param config.route.routes [array] Array containing event route configuration
+##
 config:
   logLevel: debug
   logFormat: pretty
   receivers:
     - name: "dump"
       file:
         path: "/dev/stdout"
-        # layout:
-        #   message: "{{ .Message }}"
-        #   reason: "{{ .Reason }}"
-        #   type: "{{ .Type }}"
-        #   count: "{{ .Count }}"
-        #   kind: "{{ .InvolvedObject.Kind }}"
-        #   name: "{{ .InvolvedObject.Name }}"
-        #   namespace: "{{ .Namespace }}"
-        #   component: "{{ .Source.Component }}"
-        #   host: "{{ .Source.Host }}"
+        ## layout:
+        ##   message: "{{ .Message }}"
+        ##   reason: "{{ .Reason }}"
+        ##   type: "{{ .Type }}"
+        ##   count: "{{ .Count }}"
+        ##   kind: "{{ .InvolvedObject.Kind }}"
+        ##   name: "{{ .InvolvedObject.Name }}"
+        ##   namespace: "{{ .Namespace }}"
+        ##   component: "{{ .Source.Component }}"
+        ##   host: "{{ .Source.Host }}"
```

Comment:
Since this is a code example, why are we using two `#` formatting before? Usually for code examples, we only include one. I.e.:


```
  ## Example:
  ##
  # pullSecrets:
  #   - myRegistryKeySecretName
```

Reply:
I slipped up in this case, to follow the new format it should be like in 9421ede, just like the example you point out is now
```
  ## Example:
  ## pullSecrets:
  ##   - myRegistryKeySecretName
  ##
  pullSecrets: []
```
Thanks!